pkg/proc/native/threads.go

```diff
@@ -120,22 +120,6 @@ func (thread *Thread) Stopped() bool {
 	return thread.stopped()
 }
 
-// Halt stops this thread from executing. Actual
-// implementation is OS dependant. Look in OS
-// thread file.
-func (thread *Thread) Halt() (err error) {
```

Comment:
Why remove this generic implementation in favor of copying everywhere? Also, if it must be copied, the comments need to be updated.

Reply:
Because I had to remove the defer on the linux version.